### PR TITLE
Use a distinct port for internalGETPort in tests, too

### LINKED ISSUE #2783: generic-worker tests are connecting to parent livelog, crashing it
https://community-tc.services.mozilla.com/tasks/ayBq3Tp8SiawxTzH6psUBA/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FayBq3Tp8SiawxTzH6psUBA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log
```
2020/05/07 21:30:57Z Creating task feature Mounts/Caches...
2020/05/07 21:30:57Z Creating task feature Supersede...
2020/05/07 21:30:57Z Starting task feature Live Log...
2020/05/07 21:30:57Z WARNING: Livelog failure: error 'exit status 1' and livelog output:
2020/05/07 21:30:57 input server listening... 
2020/05/07 21:30:57 listen tcp :60022: bind: address already in use
2020/05/07 21:30:58Z Starting task feature OS Groups...
[taskcluster 2020-05-07T21:30:59.394Z] Uploading redirect artifact public/logs/live.log to URL https://community-tc.services.mozilla.com/api/queue/v1/task/ayBq3Tp8SiawxTzH6psUBA/runs/0/artifacts/public/logs/live_backing.log with mime type "text/plain; charset=utf-8" and expiry 2021-05-08T21:29:33.022Z
[taskcluster:error] exit status 141
```

141 is from EPIPE, I think.

### PR DESCRIPTION
**Pull request by djmitche**
Fixes #2783.

### CODE REVIEW

File: workers/generic-worker/helper_test.go [hunk 1]
```diff
@@ -163,6 +163,12 @@ func setup(t *testing.T) (teardown func()) {
 			},
 		},
 	}
+
+	// like LiveLogGETPort and LiveLogPUTPort above, we need to use a non-default port for
+	// the livelog internalGETPort, so that we don't conflict with a generic-worker in whic
```
Comment (walac) [inline workers/generic-worker/helper_test.go:168]: nit: "whic"

**Comment by djmitche** [comment]
the generic-worker test failures now seem to be about downloading things from github :(

**Comment by djmitche** [comment]
Hm, still EPIPE failures.  @petemoore any ideas?  I think my next approach would be to combine this into a single process and use ephemeral ports..

**Comment by petemoore** [comment]
I can have a look at this on Monday.

The config should come from the setup method in `helper_test.go`.

**Comment by djmitche** [comment]
Issue's still open -- this PR might be the wrong approach though.  I'm investigating in the issue.

**Pull request closed without merging**